server/plugins/schema-plugin-api/src/main/java/com/oceanbase/odc/plugin/schema/api/DatabaseExtensionPoint.java

```diff
@@ -36,4 +37,6 @@ public interface DatabaseExtensionPoint extends ExtensionPoint {
     DBDatabase getDetail(Connection connection, String dbName);
 
     List<DBDatabase> listDetails(Connection connection);
+
+    String getCreateDatabaseSql(String databaseName, Map<String, String> parameters);
```

Comment:
this interface may be defined as follow:

```Java
DBDatabase create(Connection connection, @NonNull DBDatabase);
```

create schema by db-browser, rather than returning a sql executed by caller

Reply:
further more, `Map<String, String> parameters` may be confusing~~

Reply:
> further more, `Map<String, String> parameters` may be confusing~~

how about rename to "createDBParameters"?